flink-runtime/src/main/java/org/apache/flink/runtime/net/SSLUtils.java

```diff
@@ -55,6 +58,42 @@ public static boolean getSSLEnabled(Configuration sslConfig) {
 	}
 
 	/**
+	 * Sets SSl version and cipher suites for SSLServerSocket
+	 * @param socket
+	 *        Socket to be handled
+	 * @param config
+	 *        The application configuration
+	 */
+	public static void setSSLVerAndCipherSuites(ServerSocket socket, Configuration config) {
+		if (socket instanceof SSLServerSocket) {
+			((SSLServerSocket) socket).setEnabledProtocols(config.getString(
+				ConfigConstants.SECURITY_SSL_PROTOCOL,
+				ConfigConstants.DEFAULT_SECURITY_SSL_PROTOCOL).split(","));
```

Comment:
simply calling `split()` on a config value may lead to errors with bad error messages when the config value is misconfigured. I think it would be nice to do some more explicit handling here.